
<robot name="obstacle">
  <link name="my_obstacle">
    <inertial>
      <origin xyz="2 0 0" />
      <mass value="10.0" />
      <inertia  ixx="5.0" ixy="0.0"  ixz="0.0"  iyy="100.0"  iyz="0.0"  izz="1.0" />
    </inertial>
    <visual>
      <origin xyz="0 0 0"/>
      <geometry>
        <cylinder radius="0.1" length="7"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="2 0 1"/>
      <geometry>
        <box size="1 1 2" />
      </geometry>
    </collision>
  </link>
  <gazebo reference="my_obstacle">
    <material>Gazebo/Blue</material>
  </gazebo>
  </robot>